deploy/scripts/src/cri-orange/nino-check.ts

```diff
@@ -83,6 +83,16 @@ const csvData1: Nino[] = new SharedArray('csvDataNino', () => {
     })
 })
 
+export function setup(): void {
+  describeProfile(loadProfile)
+  const imposterstarted = false
+  if (!imposterstarted) {
+    const response = imposter.handler()
+    console.log('Response status: ', response[0].status)
+    console.log('Response body: ', response[0].body)
+  }
```

Comment:
```suggestion
  const response = imposter.handler()
  console.log('Response status: ', response[0].status)
  console.log('Response body: ', response[0].body)
```
Remove the useless variable assignment and if condition

Reply:
Done